web/styles/message_row.css

```diff
@@ -422,7 +422,8 @@
 .message_controls {
     display: flex;
     align-items: baseline;
-    font-size: 16px;
+    /* 16px at 16px/1em */
+    font-size: 1em;
```

Comment:
Is this a noop? I wonder if it can just be removed.

Reply:
Ah, yes it can. I did the em conversion just in case I discovered other font-size adjustments in the area, but there are none.

Reply:
Removed.